python/setup.cfg

```diff
@@ -22,6 +22,8 @@ keywords =
 [options]
 zip_safe = false
 python_requires = >=3.9
+install_requires =
+    deprecated
```

Comment:
I was hesitant to take on an external dep, but after reading through it, the Sphinx integration is really useful.